include/roboteam_ai/stp/Tactic.h

```diff
@@ -0,0 +1,49 @@
+//
+// Created by jessevw on 03.03.20.
+//
+
+#ifndef RTT_TACTIC_H
+#define RTT_TACTIC_H
+
+#include <vector>
+#include <roboteam_utils/containers/state_machine.hpp>
+
+#include "stp/StpInfo.h"
+#include "stp/new_skills/Rotate.h"
+#include "stp/new_skills/SetDribbler.h"
+
+namespace rtt::ai::stp {
+    class Skill;
+
+    class Tactic {
+    private:
```

Comment:
Protected. You want the derived classes to use or override these fields

Reply:
not mine